### DISCUSSION TITLE
OctoEverywhere install failed after firmwar update process

### DISCUSSION THREAD

**Discussion by jonny7737** [Guilouz/Creality-Helper-Script-Wiki#562] [Issues]
I have tried to uninstall OctoEverywhere and the error is similar to the install error:

`Running in K1 and K1 Max OS mode
Checking required system packages are installed...
System package install complete.
Checking Python Virtual Environment For OctoEverywhere...
Updating PIP if needed... (this can take a few seconds or so)
Could not fetch URL https://pypi.python.org/simple/pip/: There was a problem confirming the ssl certificate: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1108) - skipping
Requirement already up-to-date: pip in /usr/data/octoeverywhere-env/lib/python3.8/site-packages
Installing or updating required python libs...
  Could not find a version that satisfies the requirement websocket_client<1.7.99,>=1.6.0 (from -r /usr/data/octoeverywhere/requirements.txt (line 10)) (from versions: )
No matching distribution found for websocket_client<1.7.99,>=1.6.0 (from -r /usr/data/octoeverywhere/requirements.txt (line 10))`

Can't install and can't uninstall.
Any help is greatly appreciated.

**Comment by Guilouz** [comment]
Please report this on OctoEverywhere github: https://github.com/QuinnDamerell/OctoPrint-OctoEverywhere

**Comment by jonny7737** [comment]
Someone else reported this about 4 hours ago.

Thanks.

**Comment by Guilouz** [comment]
It's now fixed.
